neptune/exceptions.py

```diff
@@ -14,5 +14,22 @@
 # limitations under the License.
 #
 
-class InvalidApiToken(Exception):
-    pass
+
+class ConnectionLost(Exception):
```

Comment:
I think that it would be more convenient to have common parent for all of these exceptions, something a bit more specific than `Exception`